webdriver/newsession/basic.py

```diff
@@ -0,0 +1,56 @@
+# META: timeout=long
+
+import uuid
+
+import pytest
+
+from webdriver import error
+
+
+def test_basic(new_session):
+    resp, _ = new_session({"capabilities": {}})
+    assert set(resp.keys()) == {"sessionId", "capabilities"}
+
+
+def test_repeat_new_session(new_session):
+    resp, _ = new_session({"capabilities": {}})
+    with pytest.raises(error.SessionNotCreatedException):
+        new_session({"capabilities": {}})
+
+
+def test_no_capabilites(new_session):
+    with pytest.raises(error.InvalidArgumentException):
+        new_session({})
+
+
+def test_missing_first_match(new_session):
+    resp, _ = new_session({"capabilities": {"alwaysMatch": {}}})
+
+
+def test_missing_always_match(new_session):
+    resp, _ = new_session({"capabilities": {"firstMatch": [{}]}})
+
+
+def test_desired(new_session):
```

Comment:
There's a missing test where a "noise" key is present, but there is also a valid `capabilities` key. From the test title, it's not clear why this test should fail, and it's not tied to a part of the spec.